src/elements/digit-text-area/DigitTextArea.element.jsx

```diff
@@ -4,6 +4,12 @@ import TextField from "@material-ui/core/TextField";
 import useLayoutMaterialUi from "../../styles/material-ui/use-layout-material-ui";
 import makeStyles from "@material-ui/styles/makeStyles";
 
+const useFullFlex = makeStyles({
+    root: {
+        flex: "1"
+    }
+});
+
```

Comment:
What's the difference between this and what was before?

Reply:
I was having a performance issue in `DigitTextArea` for Gamma, I'm guessing this was the issue with calling makeStyles every render.